md/bonita-bpm-portal-interface-overview.md

```diff
@@ -1,32 +1,51 @@
+  - the **Technical User** is responsible for opening permissions to use Bonita in non-production or production environments by creating users. S·he is also the only person with permissions to start and stop the BPM services for maintenance or update (namely BDM update). For more information, go to the dedicated [documentation page](first-steps-after-setup.md).
 
-## Default profiles
+## Default User Interfaces
 
-There are three default profiles available in the Portal: **User**, **Administrator**, and (in the Efficiency, Performance and Enterprise editions) **Process manager**.
+Starting with Bonita 2021.1, there are two types of User Interfaces provided by Bonita: Bonita Portal (legacy), and Bonita Applications (the way to go).  
 
-The **Administrator** is responsible for the administration of the Portal at tenant level, and particularly for the management of the processes, the organization, the reports, the custom profiles and the Look & Feel.
+ ::: info
+ **Note:** In next versions of Bonita, Portal will be replaced with Applications, so we encourage you to use the applications as of now.
+ :::   
 
-Example of the Administrator profile interface
-![](images/images-6_0/admin_view7.1.png)<!--{.img-responsive}-->
+Bonita Portal gives access to:
+  - a User view
+  - an Administrator view
+  - a Process Manager view, with administrator rights applicable on a defined scope of the deployed processes (in Efficiency, Performance, and Enterprise editions only)
+  - a Technical User view
 
-The **Process Manager** shares process management responsibilities with the Administrator for the processes he has been declared as the Process Manager.
+Each view accounts for a set of single-page or multiple-page menu options.
+Bonita Portal views are automatically launched when running a process from the Studio. You can also open Bonita Portal with the Portal icon in the Studio Coolbar.
 
-The **User** is responsible of performing the tasks for which she is a candidate and also for starting new cases of the processes to which she has access.
+In Bonita 2021.1, Bonita Applications offer:
+  - the Bonita User Application
+  - the Bonita Administrator Application
+  
+Both Bonita Applications can be downloaded from the Welcome Page of the Studio, in the "Resources" tile.
 
-Example of the User profile interface
-![](images/user_tasklist.png)<!--{.img-responsive}-->
+Applications are more flexible than the portal, in two ways:
+   - The navigation can be customized: remove the pages that are not needed, add custom pages that are needed. This is the equivalent of Custom Profiles, but with the flexibility of an application. 
+   - Some pages of those applications have been recreated with Bonita UI Designer (see the list [here]). This means that you can open and customize the page to make it fit the precise needs of the users. Starting with an existing page accounts for a big gain of time.
```

Comment:
same comment as Dumitru: the link is missing

Reply:
indeed...